pages/deployments/oidc/oidc_with_aws.md

```diff
@@ -0,0 +1,204 @@
+    1. Ensure the `StringLike` subsection's _subject_ field name (your provider URL appended by `:sub`—`agent.buildkite.com:sub`) has at least one value that matches the format: `organization:ORGANIZATION_SLUG:pipeline:PIPELINE_SLUG:ref:REF:commit:BUILD_COMMIT:step:STEP_KEY`, where the constituent fields of this line determine the conditions (that is, when the values specified in these constituent fields have been met) under which the IAM role is granted in exchange for the OIDC token. The following constituent field's value:
+        - `ORGANIZATION_SLUG` can be obtained:
+
+            * From the end of your Buildkite URL, after accessing **Pipelines** in the global navigation of your organization in Buildkite.
+
+            * By running the [List organizations](/docs/apis/rest-api/organizations#list-organizations) REST API query to obtain this value from `slug` in the response. For example:
+
+                ```bash
+                curl - X GET "https://api.buildkite.com/v2/organizations" \
+                  -H "Authorization: Bearer $TOKEN"
+                ```
+        - `PIPELINE_SLUG` can be obtained:
+
+            * From the end of your Buildkite URL, after accessing **Pipelines** in the global navigation of your organization in Buildkite, then accessing the specific pipeline to be specified in the custom trust policy.
+
+            * By running the [List pipelines](/docs/apis/rest-api/pipelines#list-pipelines) REST API query to obtain this value from `slug` in the response from the specific pipeline. For example:
+
+                ```bash
+                curl - X GET "https://api.buildkite.com/v2/organizations/{org.slug}/pipelines" \
+                  -H "Authorization: Bearer $TOKEN"
+                ```
+        - `REF` is usually replaced with `refs/heads/main` to enforce the IAM role's access and use to only the `main` branch, `refs/tags/*` to ensure only tagged releases are able to be deployed, or a wildcard `*` if the IAM role can access and use all branches.
+        - `BUILD_COMMIT` (optional) can be omitted and if so, is usually replaced with a single wildcard `*` at the end of the line.
+        - `STEP_KEY` (optional) can be omitted and if so, is usually replaced with a single wildcard `*` at the end of the line.
+
+    **Note:** When formulating your _subject_ field's value, you can replace any of the constituent field values above with a wildcard `*` to not set limits on those constituent fields.
+
+    You can also allow this IAM role to be used with other pipelines, branches, commits and steps by specifying multiple comma-separated values for the `agent.buildkite.com:sub` _subject_ field.
+
+    For example, to allow the IAM role to be used for any pipeline in the Buildkite organization `example-org`, when building on any branch or tag, specify a single _subject_ field value of `organization:example-org:pipeline:*:ref:refs:*`.
```

Comment:
CC @tessereth can you confirm this is valid for our public docs here?

Reply:
Yep, it's correct, though it can be even simpler:

```suggestion
    For example, to allow the IAM role to be used for any pipeline in the Buildkite organization `example-org`, when building on any branch or tag, specify a single _subject_ field value of `organization:example-org:*`.
```

Reply:
Thanks @tessereth , I'll amend the docs accordingly :-)